data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.oppia.android.data.backends.gae.NetworkApiKey
+import javax.inject.Inject
+
+/**
+ * Data class for the feedback report sent by the Android app to remote storage.
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeFeedbackReport @Inject constructor(
```

Comment:
This should not be an inject constructor--it's being constructed by us & Moshi, not Dagger.

Reply:
Done